internal/profile/validation.go

```diff
@@ -70,7 +80,8 @@ func Validate(ctx context.Context, upc UserProfile, realm string, input Contains
 func ValidateUser(profile kc.UserProfileRepresentation, input ContainsFields, apiName string, checkMandatory bool) error {
 	for _, attrb := range profile.Attributes {
 		if !attrb.AnnotationMatches(apiName, func(value string) bool {
-			return strings.EqualFold(value, "true") || strings.EqualFold(value, "required")
+			// Account is the only API supporting read-only
+			return strings.EqualFold(value, "true") || strings.EqualFold(value, "required") || (apiName == "account" && strings.EqualFold(value, "read-only"))
```

Comment:
Not sure if the bridge should do this check... The bridge is a pass-through, it is the responsibility of the one doing the configuration and the one coding the self-service to do it correctly.